app/views/application/_stuff.haml

```diff
@@ -0,0 +1,3 @@
+.main
+  %div#foo
+    %span{class: "sky" } Hello
```

Comment:
Avoid defining `class` in attributes hash for static class names<br>Hash attribute should start with one space after the opening brace